### PR TITLE
fix(spec): use forgesource

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@linux.com>

### CODE REVIEW

File: fido-device-onboard.spec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Summary:        An implementation of the FIDO Device Onboard Specification writt
 
 License:        BSD 3
 URL:            %{forgeurl}
-Source:         %{url}/archive/%{commit}/%{name}-rs-%{commit}.tar.gz
+Source:         %{forgesource}
```
Comment (nullr0ute) [inline fido-device-onboard.spec:16]: You still need a tar file in that URL, I don't see a forgesource defined anywhere in the source tree using grep.

Comment (runcom) [reply fido-device-onboard.spec:16]: https://fedoraproject.org/wiki/Forge-hosted_projects_packaging_automation

Comment (nullr0ute) [reply fido-device-onboard.spec:16]: For reference the docs are at: https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_using_forges_hosted_revision_control

**Review by nullr0ute — APPROVED** [review]
LGTM

**Pull request merged**